package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "acorn": "^8.14.0",
+    "@actions/core": "^1.11.1",
```

Comment:
Why we need actions as a dep?

Reply:
For src/linter/reporters/github.mjs I would guess?

Reply:
Yep